internal/repository/rsl.go

```diff
@@ -23,21 +23,34 @@ var (
 
 // RecordRSLEntryForReference is the interface for the user to add an RSL entry
 // for the specified Git reference.
-func (r *Repository) RecordRSLEntryForReference(refName string, signCommit bool) error {
+func (r *Repository) RecordRSLEntryForReference(srcRefName, dstRefName string, signCommit bool) error {
```

Comment:
I'd make a similar options change for Record as well.

Reply:
done!

Reply:
i split them up into different opts packages to avoid a nam collision for the With... functions.